### PR TITLE
Combat and survival knife storage/inhand sprites

### PR DESCRIPTION
**Pull request by RiceMar1244**
## About the PR

Adds storage sprites for the combat knife and survival knife. Also changes the inhand sprites for both to make them distinct and more noticeable than the old sprite.

## Why / Balance

I use these alot and I wanted to see more storage visuals in general.

## Media

knife sprites

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

No changelog cause its just a small visual change.

### CODE REVIEW

**Review by UbaserB — APPROVED** [review]
sprite lgtm

File: Resources/Textures/Objects/Weapons/Melee/combat_knife.rsi/meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d",
+    "copyright": "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d, storage sprite by RiceMar1244",
```
Comment (Flareguy) [inline Resources/Textures/Objects/Weapons/Melee/combat_knife.rsi/meta.json:4]: Everything else on this PR seems fine, but it seems like the inhand sprites here aren't actually in the linked commit.
...in fact, I'm pretty sure even the object sprites aren't in the commit (or in desert rose for that matter,) but I've tried for like 30 minutes to track them down and have been unable to do so.
You should attribute yourself to having made the knife sprites. Something like:
`` "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d, storage & inhand sprites by RiceMar1244",``

This also goes for the survival knife.

File: Resources/Textures/Objects/Weapons/Melee/survival_knife.rsi/meta.json [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from /tg/station at commit https://github.com/tgstation/tgstation/commit/6cdc0503d32e249125151e369edf951e93bccd1a, storage sprite by RiceMar1244",
```
Comment (Flareguy) [inline Resources/Textures/Objects/Weapons/Melee/survival_knife.rsi/meta.json:4]: Same as above. In fact, I'm pretty sure the survival knife is *actually* from Desert Rose, so you can just copy paste the text above.

**Review by Flareguy — CHANGES_REQUESTED** [review]
Some very minor meta.json issues.

**Comment by RiceMar1244** [comment]
Yeah sure thing Flare, thanks for the review.

**Review by Flareguy — APPROVED** [review]

**Pull request merged**